README.md

```diff
@@ -3,4 +3,34 @@
 [![Build Status](https://github.com/Skyscanner/stylelint-plugin-backpack/workflows/CI/badge.svg?branch=master)](https://github.com/Skyscanner/stylelint-plugin-backpack/actions?query=workflow%3ACI)
 [![npm version](https://img.shields.io/npm/v/@skyscanner/stylelint-plugin-backpack.svg)](https://www.npmjs.com/package/@skyscanner/stylelint-plugin-backpack)
 
-Stylelint plugin to prevent magic arbitrary values and promote the use of Backpack tokens
\ No newline at end of file
+Stylelint plugin to prevent magic arbitrary values and promote the use of Backpack tokens.
+
+## Installation
+
+```
+npm install @skyscanner/stylelint-plugin-backpack --save-dev
+```
+
+## Usage
+
+Add `@skyscanner/stylelint-plugin-backpack` to your stylelint config plugins array, 
+
+```
+{
+  "plugins": ["@skyscanner/stylelint-plugin-backpack"]
+}
+```
+
+Then add rules you need to the rules list. All rules from `@skyscanner/stylelint-plugin-backpack` need to be namespaced with `backpack`.
```

Comment:
```suggestion
Next, add the rules you want to use to your list of rules. All rules from `@skyscanner/stylelint-plugin-backpack` need to be namespaced with `backpack`.
```

Reply:
I read this a few times and its starting to feel ambiguous. Happy to commit the suggestion if you feel its best, or just reword it altogether. The original line is a commonly used on within stylelint plugins, just as an FYI 😄

Reply:
Yeah I'll just dismiss my comments and leave it as is. Maybe today's heat is affecting my brain